### DISCUSSION TITLE
using this on linode domain update

### DISCUSSION THREAD

**Discussion by ghost** [qdm12/ddns-updater#275] [Q&A]
started to use this to update my domain name on linode to my local pc. 

i have this issue in my log file

2021/12/07 06:18:27 ERROR cannot create record: bad HTTP status: 400: : ,

seems like not working

**Comment by qdm12** [comment]
Hey there, sorry for the long delay.

I checked their docs and the API call looks fine to me. However I noticed the error decoding was wrong, so that's why it doesn't show why the request was a bad request (`400`). It's now fixed in ae7f10448da8716edb47a3535896f35d003c9cd6.

Would you please try

```sh
docker pull qmcgaw/ddns-updater
```

Then re-launch the container and report back what logs you have?
You can also set `LOG_LEVEL=debug` to have a bit more logs of what it's doing, which might be useful.

Thank you so much.
